plotly-dash/environment.yml

```diff
@@ -1,5 +1,9 @@
-name: myvenv
+name: plotly_workshop
+channels:
+  - conda-forge
+  - defaults
 dependencies:
-- python=3.6
-- pip:
-   - -r requirements.txt
+  - pandas
```

Comment:
Do you have an argument not to use a link to the `requirements.txt` file? My argument is that I prefer to have only one place where I define the dependencies

Reply:
I forgot to delete the `requirements.txt` from the branch.
The idea was to only use ` conda` with the `environment.yml` file and remove the `requirements.txt` alltogether.

Reply:
The solution with `requirements.txt` linked in `environment.yml` works both for conda user and for non-conda users (like me), this is why I had implemented it. If `requirements.txt` is there it should not bother the beginners too much as they would only follow the instructions.

Reply:
Ah, gotcha. Shall I restore the original version with conda and pip?

Reply:
If you don't mind yes, otherwise I can do it :)